mathml/presentation-markup/too-few-children.html

```diff
@@ -0,0 +1,455 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Test if elements layout properly if they have too few children</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#script-and-limit-schemata">
+<meta name="assert" content="Elements should fill in their children so layout is maintained.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/mathml/support/feature-detection.js"></script>
+<script src="/mathml/support/layout-comparison.js"></script>
+<script>
+  function getBox(id, i) {
+    return document.getElementById(id+i)
+  }
+
+  function removeIgnoredMrows(children) {
+    let result = [];
+    Array.from(children).forEach(child => {
+      if (child.getAttribute("class")==="ignore_mrow") {
+        if (child.childElementCount > 0)
+          Array.prototype.push.apply(result, child.children);
+      } else {
+        result.push(child);
+      }
+    });
+    return result;
+  }
+
+
+  function compareLayout(element, reference, epsilon) {
+    // Compare sizes of elements and children.
+    compareSize(element, reference, epsilon);
+    const elementBox = element.getBoundingClientRect();
+    const referenceBox = reference.getBoundingClientRect();
+
+    const elementChildren = childrenParticipatingToLayout(element);
+    const referenceChildren = removeIgnoredMrows(childrenParticipatingToLayout(reference));
+    
+    if (elementChildren.length != referenceChildren.length) {
+      throw "Reference should have the same number of children participating to layout."
+    }
+
+
+    for (let i = 0; i < elementChildren.length; i++) {
+        const childBox = elementChildren[i].getBoundingClientRect();
+        const referenceChildBox = referenceChildren[i].getBoundingClientRect();
+        compareLayout(childBox, referenceChildBox, epsilon);
+    }
+  }
+
+
+  setup({ explicit_done: true });
+  window.addEventListener("load", () => { document.fonts.ready.then(runTests); });
+
+  function runTests() {
+
+    // elements being tested
+    const elements = [
+      {name:"mover", n:2},
+      {name:"munder", n:2},
+      {name:"munderover", n:3},
+      {name:"msub", n:2},
+      {name:"msup", n:2},
+      {name:"msubsup", n:3},
+      {name:"mmultiscripts", n:1},
+      {name:"mfrac", n:2},
+      {name:"mroot", n:2},
+      {name:"msqrt", n:1},
+      {name:"mstyle", n:1},
+      {name:"merror", n:1},
+      {name:"mpadded", n:1},
+      {name:"mphantom", n:1},
+      {name:"menclose", n:1},
+      {name:"mtable", n:1},
+      {name:"mtr", n:1},
+      {name:"mtd", n:1},
+      {name:"maction", n:1},
+      {name:"semantics", n:1},
+      {name:"math", n:1}
```

Comment:
I don't think the tests for n=1 are correct. It seems to me that all these elements are allowed to have no children. We should only keep the elements that have explicit expected number of children in MathML Core.

I would also move mmultiscripts and tabular elements in separate tests.